### PR TITLE
Hebrew translation (100% complete)

### PR DESCRIPTION
**Pull request by RealA10N**
This Translation is about #224.

**files translated**:
  - [x] `def/he/about.cson`
  - [x] `def/he/context.cson`
  - [x] `def/he/menu_darwin.cson`
  - [x] `def/he/menu_linux.cson`
  - [x] `def/he/menu_win32.cson`
  - [x] `def/he/settings.cson`
  - [x] `def/he/welcome.cson`

**validation**
  - [x] I've validated my translation locally by running

`npm run validation -- --locale MY_LOCALE`

### CODE REVIEW

**Comment by RealA10N** [comment]
For some reason, every time I have tried to run the `npm run validation` command I got the same error, for a couple of files. This error message keeps appearing for almost every line in my translated files. How can I fix this error message?
`AssertionError:  
invalid or missing hotkey hint in XXX (every time for a different 'hotkey hint')`

File: spec/locales.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const LOCALES = [
   'uk',
   'zh-cn',
   'zh-tw',
+  'he',
```
Comment (liuderchi) [inline spec/locales.js]: Please put `'he'` in alphabetical order

File: package.json [hunk 2]
```diff
@@ -129,6 +129,10 @@
           "value": "zh-tw",
           "description": "繁體中文 (zh-tw)"
         },
+        {
+          "value": "he",
```
Comment (liuderchi) [inline package.json]: Please put `'he'` in alphabetical order

File: spec/locales.js (hunk 1 above)
Comment (liuderchi) [inline spec/locales.js]: Please put `'he'` in alphabetical order

File: package.json (hunk 2 above)
Comment (RealA10N) [reply package.json]: Thanks! Fixed in the last commit.

File: spec/locales.js (hunk 1 above)
Comment (RealA10N) [reply spec/locales.js]: Fixed in the last commit

**Pull request merged**